tools/repl/src/repl.rs

```diff
@@ -81,6 +81,7 @@ impl REPL {
             Some("e") | Some("envs") => self.show_envs(args),
             Some("f") | Some("fs") => self.show_fs(args),
             Some("i") | Some("interface") => self.show_interface(),
+            Some("h") | Some("heap") => self.show_heap_statistic(),
```

Comment:
h is usually for `help`. How about `s` for `stats`? And maybe also add `h` to show all possible commands?